.github/workflows/make_release.yml

```diff
@@ -0,0 +1,17 @@
+name: Make a Github release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          prerelease: true
```

Comment:
Hmm, any chance you could change this to be conditional, and set prerelease if there's `-rc` in the release name? I don't think we want to set the tag to pre-release for final releases. It is fine if we add the notes later, editing an existing tag.

Reply:
good call. I'll check it out

Reply:
![image](https://user-images.githubusercontent.com/81654/229182786-21e62bf2-4df9-4fed-b067-68165e06598b.png)